libs/components/progress-indicator/src/lib/modules/progress-indicator/progress-indicator-status-marker/progress-indicator-status-marker.component.scss

```diff
@@ -61,10 +159,13 @@ $sky-progress-indicator-success-color: #5a9936;
 
         &:before {
           content: '';
-          display: block;
+          display: var(
+            --sky-override-progress-indicator-status-marker-before-display,
+            none
+          );
           width: 5px;
           height: 1px;
-          background-color: $sky-progress-indicator-success-color;
+          background-color: #5a9936;
```

Comment:
is this color something that can be tokenized?

Reply:
This is wizard so it is not tokenized at this time.